src/app/zap-templates/zcl/data-model/chip/service-area-cluster.xml

```diff
@@ -49,25 +49,25 @@ limitations under the License.
 
   <struct name="ProgressStruct">
     <cluster code="0x0150"/>
-    <item name="LocationID"             type="int32u"                 optional="false"/>
+    <item name="AreaID"                 type="int32u"                 optional="false"/>
     <item name="Status"                 type="OperationalStatusEnum"  optional="false"/>
     <item name="TotalOperationalTime"   type="elapsed_s"              optional="true" isNullable="true"/>
     <item name="EstimatedTime"          type="elapsed_s"              optional="true" isNullable="true"/>
   </struct>
 
-  <enum name="SelectLocationsStatus" type="enum8">
+  <enum name="SelectAreasStatus" type="enum8">
     <cluster code="0x0150"/>
     <item value="0x00" name="Success"/>
-    <item value="0x01" name="UnsupportedLocation"/>
-    <item value="0x02" name="DuplicatedLocations"/>
+    <item value="0x01" name="UnsupportedArea"/>
+    <item value="0x02" name="DuplicatedAreas"/>
     <item value="0x03" name="InvalidInMode"/>
     <item value="0x04" name="InvalidSet"/>
   </enum>
 
-  <enum name="SkipCurrentLocationStatus" type="enum8">
+  <enum name="SkipAreaStatus" type="enum8">
     <cluster code="0x0150"/>
     <item value="0x00" name="Success"/>
-    <item value="0x01" name="InvalidLocationList"/>
+    <item value="0x01" name="InvalidAreaList"/>
     <item value="0x02" name="InvalidInMode"/>
```

Comment:
This is missing InvalidSkippedArea.  Separate PR for that, I guess?

Reply:
Correct. I was going to add it here but I thought I should only do name changes in this one. This should be added when fixing [this issue](https://github.com/project-chip/connectedhomeip/issues/34670).